docs/integrations/fastapi.md

```diff
@@ -102,6 +102,40 @@ To avoid tracing certain URLs, you can specify a string of comma-separated regex
 - The environment variable `OTEL_PYTHON_FASTAPI_EXCLUDED_URLS`.
 - The environment variable `OTEL_PYTHON_EXCLUDED_URLS` (which will also apply to other instrumentation).
 
+!!! example
+    If you'd like to trace all URLs except the base `/` URL, you can use the following regex for `excluded_urls`: `^https?://[^/]+/$`
+
+    Breaking it down:
+
+    * `^` matches the start of the string
+    * `https?` matches `http` or `https`
+    * `://` matches `://`
+    * `[^/]+` matches one or more characters that are not `/`
+    * `/` matches `/`
+    * `$` matches the end of the string
```

Comment:
```suggestion
    * `^` matches the start of the string
    * `https?` matches `http` or `https`
    * `://` matches `://`
    * `[^/]+` matches one or more characters that are not `/` (this will be the host part of the URL)
    * `/` matches `/`
    * `$` matches the end of the string
    
    So this regex will only match routes that have no path after the host.
```